examples/simple/src/posts/PostCreate.tsx

```diff
@@ -94,18 +135,44 @@ const backlinksDefaultValue = [
         url: 'http://google.com',
     },
 ];
+
+const useMutationMode = (): MutationMode => {
+    const [searchParams] = useSearchParams();
+    const mutationMode = searchParams.get('mutationMode') ?? 'pessimistic';
+
+    return ['optimistic', 'undoable', 'pessimistic'].includes(mutationMode)
+        ? (mutationMode as MutationMode)
+        : 'pessimistic';
+};
+
 const PostCreate = () => {
     const defaultValues = useMemo(
         () => ({
             average_note: 0,
         }),
         []
     );
+    const mutationMode = useMutationMode();
     const dateDefaultValue = useMemo(() => new Date(), []);
     return (
-        <Create redirect="edit">
+        <Create
+            redirect="edit"
+            mutationMode={mutationMode}
+            transform={data => ({ ...data, id: getNewId(mutationMode) })}
+        >
+            <Alert severity="info" role="presentation">
```

Comment:
Please hide this alert when not in offline mode.

Reply:
Those are instructions to know how to test the mutation modes. Why do you want to hide them?